test/uk/gov/hmrc/childcarecalculatorfrontend/views/PartnerMinimumEarningsViewSpec.scala

```diff
@@ -36,16 +36,18 @@ class PartnerMinimumEarningsViewSpec extends NewYesNoViewBehaviours {
   def createViewWithAmount = (amount: BigDecimal) => view(frontendAppConfig, form, NormalMode, amount)(fakeRequest, messages)
 
   "PartnerMinimumEarnings view" must {
-    behave like normalPageWithTitleAsString(
-      view = createView,
-      messageKeyPrefix = messageKeyPrefix,
-      messageKeyPostfix = "",
-      title = messages("partnerMinimumEarnings.heading", 0),
-      heading = Some(""),
-      expectedGuidanceKeys= Seq(),
-      args = 0
-    )
```

Comment:
I don't understand why did you remove this section, only to copy-paste 2 tests from it.